QUEST_LV_0200.tsv

```diff
@@ -3339,30 +3339,30 @@ QUEST_LV_0200_20150323_003338	So we created the incident.{nl}Our actions.. I am
+QUEST_LV_0200_20150323_003343	$The only thing I could tell to the villagers who were wounded was to wait..{nl}Even if it was the monsters' doing. Reality is cruel.
 QUEST_LV_0200_20150323_003344	$A thorn forest.. If it weren't for you, this place would have became just like that.{nl}I feel grateful to the villagers since I believe they guided you to me.
 QUEST_LV_0200_20150323_003345	The seal here was able to be protected by you.{nl}But, I can't relax yet because the seal of Spring Light Wood is still left.
 QUEST_LV_0200_20150323_003346	My sister Laima has told me about this in the past.{nl}The Goddesses have a duty to save humans, but the humans also have a duty to save us.
 QUEST_LV_0200_20150323_003347	Farmer Druva
-QUEST_LV_0200_20150323_003348	I don't care about the monsters or the seals.{nl}I don't worry about frosts. You understand right?
-QUEST_LV_0200_20150323_003349	We should grab a chance when times are like this.{nl}The prices of agricultural products are increasing due to monsters.
+QUEST_LV_0200_20150323_003348	$I don't care about the monsters or the seals.{nl}Someone else getting hurt is their problem. We clear?
+QUEST_LV_0200_20150323_003349	$Times like this are an opportunity.{nl}Prices for agricultural products are increasing due to the monsters.
 QUEST_LV_0200_20150323_003350	$I was a refugee from the capital.{nl}I moved here hearing that this place was safe.. but that apparently is untrue. I'm ruined.
-QUEST_LV_0200_20150323_003351	Ahah, I am hungry.. I can't endure anymore.{nl}Hey. Is there something to eat?
-QUEST_LV_0200_20150323_003352	I can't do anything since I am starving to death..
-QUEST_LV_0200_20150323_003353	I came here to save my son who is cursed.{nl}If you see a young man who is about to die due to hunger, let me know.
+QUEST_LV_0200_20150323_003351	$Ahah, I am so hungry.. I can't endure any longer.{nl}Hey. Is there something to eat?
+QUEST_LV_0200_20150323_003352	$I can't do anything since I am starving to death..
+QUEST_LV_0200_20150323_003353	$I came here to save my son who is cursed.{nl}If you see a young man who is about to starve to death, let me know.
 QUEST_LV_0200_20150323_003354	{memo X}I will return to my hometown with my son.{nl}Yes. The Revelator of the Goddesses helped us.
 QUEST_LV_0200_20150323_003355	{memo X}I want to see my son being healthy again.{nl}It's too painful to wait until my son to throw up the vicious energy.
 QUEST_LV_0200_20150323_003356	What should I do about it to save the spirits?
 QUEST_LV_0200_20150323_003357	These are not Maven's lessons.{nl}They are just Naktis' curse.
-QUEST_LV_0200_20150323_003358	We have four more necklaces besides this necklace, each with a lesson of Maven.{nl}They are all useless to me.
+QUEST_LV_0200_20150323_003358	$There are four more necklaces besides this necklace, each with a lesson of Maven.{nl}They are all meaningless to me.
```

Comment:
lesson? Does it mean the same as 'teaching', because that's what the original refers to.